company/team/org_chart.md

```diff
@@ -29,10 +29,16 @@ The org chart is generated automatically. [Need to edit it?](#how-to-edit)
 <!-- When updating the ops team list below, please also update handbook/index.md. -->
 ### [People ops](../../handbook/people-ops/index.md#people-ops-team-members)
 ### [Business & product operations](../../handbook/ops/bizops/index.md#members)
-### [Sales strategy & operations](../../handbook/ops/sales-ops/index.md#members)
 ### [Finance & accounting](../../handbook/ops/finance/index.md#members)
 ### [Legal](../../handbook/ops/legal/index.md#members)
 
+
+## Sales
+<!-- When updating the engineering team list below, please also update handbook/index.md. -->
+### [Sales team](../../handbook/handbook/sales#members)
+### [SDR team](../../handbook/sales/sdrteam#members)
+### [Sales strategy & operations](../../handbook/ops/sales-ops/index.md#members)
```

Comment:
Root for the Sales ops team is Ops and not Sales, which doesn't impact the look or meaning of the page, but is a minor issue that we might want to clean up,